app/models/ticket.rb

```diff
@@ -48,6 +48,13 @@ def self.total_price(conference, user)
     result ? result : Money.new(0, 'USD')
   end
 
+  def self.tickets_turnover(ticket_id, conference_id)
```

Comment:
Ever heard from instance methods ;)
http://www.railstips.org/blog/archives/2009/05/11/class-and-instance-methods-in-ruby/